pkg/volume/util/goroutinemap/goroutinemap.go

```diff
@@ -0,0 +1,252 @@
+/*
```

Comment:
Call this something besides "goroutinemap" It is very specific. Like maybe "AttachMountDependencyTracker" Or something more pithy.